texar/utils/utils.py

```diff
@@ -605,6 +608,37 @@ def get_instance_kwargs(kwargs: Kwargs, hparams: ParamDict) -> Kwargs:
     return kwargs_
 
 
+def get_output_size(instance_: Optional[Module]) -> Optional[int]:
```

Comment:
It's strange to accept a `None` as argument. Are there any use cases where this would be more convenient?